superset-frontend/src/components/Select/AsyncSelect.tsx

```diff
@@ -527,7 +527,7 @@ const AsyncSelect = forwardRef(
             )
           }
           oneLine={oneLine}
-          tagRender={oneLine ? oneLineTagRender : undefined}
+          tagRender={customTagRender}
```

Comment:
Previously, we only used a custom tag renderer if `oneLine` was true. Does `customTagRender` preserve the same behavior?

Reply:
yes `customTagRender` is the same  as `oneLineTagRender`. It has the same behaviour except it does not render the select all tag.

Reply:
What I mean to ask is if `customTagRender` has the same behavior as `tagRender={undefined}`

Reply:
Oh I see. The only difference with `customTagRender` than the default is the inclusion of the `onMouseDown` handler that prevents the dropdown from opening when clicking the tag icons. This was added for the `oneLine` mode and it seemed useful for the regular use case, but i can add a check to not include it when the component is not in `oneLine` mode?

Reply:
Interesting. I like the behavior. No change is needed then.